src/styles/card.scss

```
@@ -0,0 +1,122 @@
+.cards {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  grid-template-rows: repeat(2, 408px);
+  gap: 48px 46px;
+  box-sizing: border-box;
+  width: 1024px;
+  margin: 0 auto;
+  justify-content: center;
+  padding: 50px 40px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", Helvetica, sans-serif;
+  transition: all 300ms ease-in-out;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__icon {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &:hover &__title {
+    color: $card-title-hover;
+  }
+
+  &__subtitle {
+    margin: 0 0 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__rewiews {
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-color;
+  }
+
+  &__rewiews-text {
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    margin: 0;
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary;
+  }
+
+  &__price-coast {
+    margin: 0;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
```

Need to check and fix it everywhere
```suggestion
  &__price {
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin-bottom: 16px;
    
    &-text {}
    
    &-coast {}
  }
```